src/main/java/task/Deadline.java

```diff
@@ -0,0 +1,46 @@
+package task;
+
+import java.time.LocalDate;
+import java.time.LocalDateTime;
+import java.time.YearMonth;
+import java.time.format.DateTimeFormatter;
+
+/**
+ * Class that stores a Task that contains a date(by).
+ */
+public class Deadline extends Task {
+
+    protected LocalDate deadlineDate = null;
```

Comment:
Is there a reason why you left out the documentation for this and the older classes?

Reply:
Whoops, I seem to have missed adding the necessary documentation of the task package. Thanks for pointing it out, I've added in the documentation.